### DISCUSSION TITLE
where to set "Displays compass bearing marks along the horizon"?

### DISCUSSION THREAD

**Discussion by bestive** [Stellarium/stellarium#658] [Q&A] [answered]
I 'll "Displays compass bearing marks along the horizon",where is it ? or config.ini  to set?
tks! 
nice!

**Comment by alex-w** [comment, marked as answer]
Please use Compass Marks plugin for bearing marks along the horizon (enabling through GUI or config.ini).

**Comment by bestive** [comment]
@alex-w  thank you very much!
Yesterday I have solved, find the compass plugin.
I really like Stellarium, it's great!
